src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,33 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.InvalidUserException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final String LOGIN_ALREADY_TAKEN = "This login is already taken";
+    private static final String LOGIN_TOO_SHORT = "Login cannot be null or less than 6 characters";
+    private static final String PASSWORD_TOO_SHORT
```

Comment:
There is a missing semicolon at the end of the `PASSWORD_TOO_SHORT` constant declaration. This will cause a syntax error.